src/board/generic_esp32/generic-esp32-board.c

```diff
@@ -278,22 +399,71 @@ void RtcStopAlarm( void )
 
 static RadioOperatingModes_t OperatingMode;
 
+void SX126xWaitOnBusy( void )
+{
+    while(gpio_get_level(lora_spi.busy) == 1);
+}
+
 /*!
  * \brief HW Reset of the radio
  */
 void SX126xReset( void )
```

Comment:
I don't know the reason, but in

https://github.com/manuelbl/ttn-esp32/blob/master/src/hal/hal_esp32.c

there is an option to keep reset floating

Reply:
I'm not sure of the reason either. Searching online it seems to be a bad idea to leave reset pin floating/doesn't make any difference, so I think I will just leave it disabled.